### PR TITLE
Feature/unifying connectors

### PR DESCRIPTION
**Pull request by MHove89**

### CODE REVIEW

File: umbraco-marketplace-enterspeed.source.umbracocms.v11.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿{
```
Comment (jesperweber) [inline umbraco-marketplace-enterspeed.source.umbracocms.v11.json:1]: The marketplace files are automatically picked up by Umbraco when a filename in the root of the default branch matches the NuGet package name. (https://marketplace.umbraco.com/listing)

So if our new .net core package gets the name of Enterspeed.Source.UmbracoCms, we should add a file called umbraco-marketplace-enterspeed-source-umbracocms.json and we properbly don't need the v11 version of the file. 

If I'm allowed to push to your branch I could also do it. There might be some other changes, like I don't think we would have the v9 and v10 packages as related packages as they would be deprecated.

File: tests/Enterspeed.Source.UmbracoCms.V10.Tests/Services/UmbracoRedirectsServiceTests.cs [hunk 2]
```diff
@@ -1,106 +1,106 @@
...
-            // Act
-            var redirect = sut.GetRedirects(guid, "en-US")[0];
-
-            // Assert
-            Assert.Equal(expected, redirect);
-        }
-    }
-}
\ No newline at end of file
+﻿//using System;
```
Comment (jesperweber) [inline tests/Enterspeed.Source.UmbracoCms.V10.Tests/Services/UmbracoRedirectsServiceTests.cs:1]: Can we restore the tests?

**Review by jesperweber — APPROVED** [review]
The code is looking good and I have tested in multiple umbraco versions with a local nuget package and it's working :-)

Only one comment about the tests.

**Review by jesperweber — APPROVED** [review]
Tested and looking good :-)

**Comment by MHove89** [comment]
I need 1 approving reviewer. @jesperweber and I have tested everything quite significantly. I need minimum 1 approver to be able to merge to develop. 

@emilras  Could you ensure that this gets approved? :)

**Review by emilras — APPROVED** [review]
Proforma approval to get things moving!

**Review by jesperweber — APPROVED** [review]

**Pull request merged**